src/App.tsx

```diff
@@ -1,9 +1,57 @@
+import React from 'react';
+import { Routes, Route, NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+import * as PeopleTable from './components/PeopleTable';
 import './App.scss';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+export const App: React.FC = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }): string => {
+    return isActive ? 'is-info' : 'is-light';
+  };
+
+  return (
+    <div className="App">
+      <nav className="buttons m-5">
+        <NavLink
+          to="/"
+          className={(param) => classNames('button', getLinkClass(param))}
+        >
+          Home
+        </NavLink>
+        <NavLink
+          to="/people"
+          className={(param) => classNames('button', getLinkClass(param))}
+        >
+          People
+        </NavLink>
+      </nav>
+
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <h1 className="title is-3 ml-5">
+              Home page
+            </h1>
+          )}
+        />
```

Comment:
Use [Navigate](https://reactrouter.com/docs/en/v6/components/navigate)) component to redirect from /home to /;